jobs/mobile.md

```diff
@@ -0,0 +1,51 @@
+# Mobile Engineer
+
+## About Us
+
+Teleport is the Infrastructure Identity Company, modernizing identity, access,
+and policy for infrastructure, improving engineering velocity and resiliency of
+critical infrastructure against human factors and/or compromise.
+
+We are a fast-growing, well-funded Y-Combinator company. We value our craft,
+are strong supporters of work/life balance, and embrace a culture of humility,
+honesty, and transparency.
+
+## About this job
+
+We are looking for product-focused mobile engineers to help launch our first
+mobile app.
+
+You will build mobile apps, backend APIs, and integrations for all parts of Teleport.
+
+You will work on the full product lifecycle from working with customers to
+understand their problems, writing RFDs for your proposed design,
+implementation, testing, and documentation.
+
+Teleport is an open core project and most of the code you will write will be
+open source.
+
+## Projects you may work on
+
+* Extending Teleport's [Device
+  Trust](https://goteleport.com/docs/identity-governance/device-trust/)
+  functionality to iOS and other mobile platforms.
+* Making Teleport easier to use on mobile platforms
+* Building mobile features that support our passwordless vision
+
+## Requirements
+
+* iOS experience
+* Experience with Go, Android, or Cryptography engineering is a plus
+
+As we are a startup, you should be comfortable changing the area of focus and
+working directly with customers.
+
+## Interview Process
+
+What to expect once you apply:
+
+* You will join 30 minute intro call and we will walk you through the
+  compensation, interview process and requirements.
+* You join a Slack channel and submit a coding challenge using GitHub.
```

Comment:
```suggestion
* You’ll join a Slack channel where we’ll share a practical coding challenge to complete and submit via GitHub
```

should we speak about the length of the challenge? im not sure if thats changing, re: rj's talk yesterday